JavaScript/colorization-demo/index.html

```diff
@@ -119,7 +119,7 @@ <h4>Easily Add Color to Your Black and White Photos</h4>
 			<div class="col-md-8">
 				<p>Bring new life to old photos by automatically colorizing them using the Algorithmia API. It's as easy as pasting in a URL above.</p>
 				<p>We're excited to introduce <a href="http://blog.algorithmia.com/2016/07/cloud-hosted-deep-learning-models/" target="_blank">cloud hosted deep learning models</a> on Algorithmia. The <a href="https://algorithmia.com/algorithms/deeplearning/ColorfulImageColorization" target="_blank">Colorful Image Colorization microservice</a> is a <a href="https://algorithmia.com/blog/introduction-to-computer-vision" target="_blank">computer vision</a> algorithm trained on a million images from the Imagenet dataset.</p>
-				<p>This microservice is an implementation of the <a href="http://richzhang.github.io/colorization/" target="_blank">Colorful Image Colorization algorithm created by Richard Zhang, Phillip Isola, and Alexei A. Efros</a>.<!-- and <a href="#">how we built this demo</a>-->.</p>
+				<p>This <a href="https://algorithmia.com/blog/introduction-to-microservices" target="_blank">microservice</a> is an implementation of the <a href="http://richzhang.github.io/colorization/" target="_blank">Colorful Image Colorization algorithm created by Richard Zhang, Phillip Isola, and Alexei A. Efros</a>.</p>
```

Comment:
Since this is also linking to an external page, let's set `rel="noopener noreferrer"` for security purposes.

Reply:
Added